torchci/components/metrics/panels/TimeSeriesPanel.tsx

```diff
@@ -289,13 +356,65 @@ export default function TimeSeriesPanel({
     granularity,
     groupByFieldName,
     timeFieldName,
-    yAxisFieldName
+    yAxisFieldName,
+    false,
+    smooth,
+    sort_by,
+    chartType,
+    filter
   );
 
+  // If we have too many series, we'll only show the top N series by total value
```

Comment:
Does it need to check sort_by or can we just assume that people will not set max_items_in_series?

Reply:
it's up to the user of the component to decide, if they want max_items_in_series=5 with a random sort, why not? By default max_items_in_series=0 so will not limit